src/main.js

```diff
@@ -11,18 +11,22 @@ Vue.use(VueAxios, axios);
 axios.defaults.baseURL = "https://developers.zomato.com/api/v2.1";
 axios.defaults.headers.common["user-key"] = process.env.VUE_APP_API_KEY;
 
-//interceptors
-
 axios.interceptors.response.use(
   response => {
-    if (response.status === 403) {
-      console.log(response);
+    if (process.env.VUE_APP_API_KEY) {
+      console.log(response.status);
     }
     return response;
   },
   error => {
-    if (error.response.status === 403) {
-      alert("please use your API key to run the app");
+    if (!process.env.VUE_APP_API_KEY) {
+      alert(
+        "Please make a file named '.env.local'. Create a global variable VUE_APP_API_KEY and initialize it with your own key in this file. " +
+          error
+      );
+    }
+    if (process.env.VUE_APP_API_KEY && error.response.status === 403) {
+      alert("Your key expired or is invalid  " + error);
```

Comment:
Could provide a suggested solution. A link to Zomato's page with the description of how to get a key would be very appreciated by the user who got it all wrong (e.g. entered their github key instead of zomato key).